docs/setup_visualstudio.md

```diff
@@ -249,7 +249,7 @@ Automate user input by putting it in a file.
 ```
 Drew
 ```
-{: data-title="main_test.in" data-highlight="1" }
+{: data-title="main_test.in" }
```

Comment:
I think that all three of the IDE tutorials contain this.  I realize that its weird to highlight one line, but in context, it provides a queue of where a line in a previous block relates to:

<img width="884" alt="Screen Shot 2022-12-27 at 9 25 34 AM" src="https://user-images.githubusercontent.com/7503005/209680401-387ad23c-a5ba-4a2c-81b8-2fc8f65818a4.png">

Reply:
Yeah... I couldn't add magic comments to the other two because:
- The `.in` file doesn't have syntax-highlighting (so no way to add "comments")
- The `console` syntax highlighting doesn't have a good way to mark a full-line comment without including the prompt :(

The existing `data-highlight="..."` syntax still works, so we can leave the other code blocks as-is.